backend/src/main.ts

```diff
@@ -53,20 +54,25 @@ const run = async () => {
   const queue = queueServiceFactory(appCfg.REDIS_URL);
   const keyStore = keyStoreFactory(appCfg.REDIS_URL);
 
-  const server = await main({ db, auditLogDb, smtp, logger, queue, keyStore });
+  const pkcs11Module = initializePkcs11Module();
+  pkcs11Module.initialize();
+
+  const server = await main({ db, auditLogDb, hsmModule: pkcs11Module.getModule(), smtp, logger, queue, keyStore });
   const bootstrap = await bootstrapCheck({ db });
```

Comment:
Can we also do the HSM is working check in bootstrap as well.

Like if HSM is not configured it could say skipping HSM check due to not configured and if configured does a connection check.

Reply:
We do this in the `startService` handler for the HSM. I think this makes sense as-is, because if the startService function is run, we want to make sure all the cryptographic functions are available before attempting to use the HSM in the KMS `startService` handler